### ISSUE TITLE
Editor: tapping Android back button leaves post when media modal is active

### ISSUE THREAD

**Issue by designsimply** [Automattic/wp-calypso#1233]
Raised by @mjangda 

Android 5.1 / Chrome 45.0.2454.84 / Moto X (2nd gen)

This is a weird Android-only quirk. I was writing a post and opened the media modal via the "Add Media" button. I changed my mind and instinctively tapped the native Android back button instead expecting that it would close the modal and return me to the post. However, I was bumped back to the screen I was on before starting the post.

**Comment by brbrr** [comment]
Closing due to inactivity and low priority issue.
